changelogs/changelog.yaml

```
@@ -762,3 +762,53 @@ releases:
       name: helm_pull
       namespace: ''
     release_date: '2023-01-24'
+  3.0.0:
+    changes:
+      breaking_changes:
+      - Remove support for ansible-core < 2.14
+      - Update python kubernetes library to 24.2.0 , helm/kind-action to 1.8.0, kubernetes
+        >= 1.24.
+      bugfixes:
+      - helm - Put the chart_ref into quotes when running ``helm show chart``, ``helm
+        upgrade`` and ``helm dependency update`` commands (https://github.com/ansible-collections/kubernetes.core/issues/653).
+      - helm - delete temporary file created when deploying chart with option release_values
```

For the other releases we've been going through the new changelog entries and making sure that all options are monospace formatted like below, do we want to do that for this release too?

```suggestion
      - helm - delete temporary file created when deploying chart with option ``release_values``
```